### PR TITLE
ssl cert automation for smtp relay

### PR DESCRIPTION
**Pull request by treyx**
WIP do not merge

### CODE REVIEW

File: smtp-relay-automation.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  ENVIRONMENT="prod"
+  /root/.acme.sh/acme.sh --issue --dns dns_aws -d smtp-relay.guild-cloud.com -d smtp-relay.guild-cloud.com
+elif [[ $ACCOUNT -eq 477873552632 ]]
+then
+  DOMAIN="dev-guild-cloud.com"
+  ENVIRONMENT="dev"
+   /root/.acme.sh/acme.sh --issue --dns dns_aws -d smtp-relay.dev-guild-cloud.com -d smtp-relay.dev-guild-cloud.com
+else
+  echo "unable to identify AWS account"
+  exit 1
```
Comment (treyx) [inline smtp-relay-automation.sh:17]: I added this in, not sure if it was the best way to handle it

File: deploy.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -e
+
+STAGE=$1
+
+echo "Deploying for stage $STAGE"
```
Comment (qhartman) [inline deploy.sh:5]: Noteworthy that this just uploads a fresh docker image, it doesn't really "deploy"?

**Review by qhartman — APPROVED** [review]
LGTM

File: deploy.sh (hunk 2 above)
Comment (treyx) [reply deploy.sh:5]: I added a task in the followup card for this. It's just meant to update the image in ECR. We could have it update the stack as well

**Pull request merged**